### PR TITLE
[es] Update KubeCon dates for 2024

### PR DESCRIPTION
**Pull request by Arhell**
en PR https://github.com/kubernetes/website/pull/44158/files
/assign @ramrodo

### CODE REVIEW

**Review by dipesh-rawat — COMMENTED** [review]
Proposed change to add the 2024 Kubecon dates looks in sync with 'en' PR https://github.com/kubernetes/website/pull/44158/files

LGTM

[Preview Page](https://deploy-preview-44339--kubernetes-io-main-staging.netlify.app/es/) | [Original Page](https://kubernetes.io/es/)

File: content/es/_index.html [hunk 1]
```diff
@@ -43,12 +43,12 @@ <h2>El desafío de migrar más de 150 microservicios a Kubernetes</h2>
         <button id="desktopShowVideoButton" onclick="kub.showVideo()">Ver vídeo</button>
         <br>
         <br>
-        <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/" button id="desktopKCButton">Asista a la KubeCon en Norte América del 24 al 28 de Octubre 2022</a>
+        <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/" button id="desktopKCButton">Asista a la KubeCon en Europa del 19 al 22 de Marzo 2024</a>
```
Comment (ramrodo) [inline content/es/_index.html:46]:
```suggestion
        <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/" button id="desktopKCButton">Asiste a la KubeCon en Europa del 19 al 22 de Marzo 2024</a>
```

File: content/es/_index.html [hunk 2]
```diff
@@ -43,12 +43,12 @@ <h2>El desafío de migrar más de 150 microservicios a Kubernetes</h2>
...
         <br>
         <br>
         <br>
         <br>
-        <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-europe-2023/" button id="desktopKCButton">Asista a la KubeCon en Europa del 17 al 21 de Abril 2023</a>
+        <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america-2024/" button id="desktopKCButton">Asista a la KubeCon en Norte América del 12 al 15 de Noviembre 2024</a>
```
Comment (ramrodo) [inline content/es/_index.html:51]:
```suggestion
        <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america-2024/" button id="desktopKCButton">Asiste a la KubeCon en Norteamérica del 12 al 15 de Noviembre 2024</a>
```

**Review by ramrodo — CHANGES_REQUESTED** [review]
Thanks!

Just some suggestions more related to our language.

File: content/es/_index.html (hunk 2 above)
Comment (electrocucaracha) [reply content/es/_index.html:51]: Great catch, I have to investigate it on the [RAE](https://www.rae.es/dpd/Norteam%C3%A9rica)

File: content/es/_index.html (hunk 1 above)
Comment (Arhell) [reply content/es/_index.html]: Done, thanks

File: content/es/_index.html (hunk 2 above)
Comment (Arhell) [reply content/es/_index.html]: Done, thanks

**Comment by electrocucaracha** [comment]
@ramrodo could you double-check this, it seems to be ready to be merged

**Comment by ramrodo** [comment]
Thanks! It's ready.
/lgtm
/approve

**Pull request merged**